spec/support/factory_bot.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
```

Comment:
Why not just put this in spec helper?

Reply:
Yeah, I was kind of thinking about that when I copied this over. I originally had it separate because I think I followed [the documentation](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#configure-your-test-suite). but I can move it